src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableProvider/DefaultCloudTableProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2014 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Threading;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Serilog.Sinks.AzureTableStorage.AzureTableProvider
+{
+    public class DefaultCloudTableProvider : ICloudTableProvider
+    {
+        private readonly int _waitTimeoutMilliseconds = Timeout.Infinite;
```

Comment:
Nit - the codebase doesn't use explicit `private` or `internal`

Reply:
Cool - will remove explicit uses of private and internal.